cpu/sensors.py

```diff
@@ -68,36 +67,23 @@ def setUp(self):
             if not s_mg.check_installed("lm_sensors") and not s_mg.install(
                     "lm_sensors"):
                 self.skip('Need sensors to run the test')
-        if d_distro.arch in ["ppc64", "ppc64le"]:
-            if 'platform\t: PowerNV\n' not in cpu._get_cpu_info():
-                self.skip('sensors test is applicable to bare-metal environment.')
-            kernel_ver = platform.uname()[2]
-            l_config = "CONFIG_SENSORS_IBMPOWERNV"
-            config_op = process.system_output(
-                'cat /boot/config-' + kernel_ver +
-                '| grep -i --color=never ' + l_config, shell=True)
-            if "=" not in config_op:
+
+        if 'platform\t: PowerNV\n' not in cpu._get_cpu_info():
```

Comment:
The PowerNV check has to be only for the arch `"ppc64", "ppc64le"`. Hence this would always skip on x86.
It should be
```
        if d_distro.arch in ["ppc64", "ppc64le"]:
            if 'platform\t: PowerNV\n' not in cpu._get_cpu_info():
                self.skip('sensors test is applicable to bare-metal environment.')
            else:
                config_check = linux_modules.check_kernel_config('CONFIG_SENSORS_IBMPOWERNV')
                if config_check == 0:
                ...
                ...
                else:
                    self.log.info('Module is Built In')
```
as all of this is specific to power architecture